js/test/npmJsonRpc.js

```diff
@@ -15,13 +15,13 @@
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
 try {
-  var JsonRpc = require('../.npmjs/jsonrpc/library.js').default;
+  const JsonRpc = require('../npm/jsonrpc').default;
 
   if (typeof JsonRpc !== 'object') {
     throw new Error('JsonRpc');
   }
 
-  console.log(JsonRpc);
+  console.log('JSON RPC Endpoints:', Object.keys(JsonRpc));
```

Comment:
Think the original was actually a debug that slipped through, however this makes (slightly) more sense.